### ISSUE TITLE
Feature request: able to paste link to note w/o title

### ISSUE THREAD

**Issue by skotzko** [drgrib/alfred-bear#47]
I find myself using the plugin all the time to link to my notes in Bear from other places (e.g. task mgmt tool, journal). Often, I have anchor text I want to use for the link that is different than the note title. It would be useful to be able to paste just the note x-callback URL, without the title. 

## Example
I was just linking from my task mgmt software to a note in Bear for a book called "Pivot." I had existing anchor text that I wanted to link to the note, which was "this is from the Pivot book I first read in 2017." 

When I pasted the link, I got: `[Pivot: The Only Move That Matters Is Your Next One](bear://x-callback-url/open-note?id=D65B06EC-2AE9-40D0-9BB2-D6D5580F768E-1862-00001BFEDC390C09&show_window=yes&new_window=yes)`

What I would love is a way to just paste: `bear://x-callback-url/open-note?id=D65B06EC-2AE9-40D0-9BB2-D6D5580F768E-1862-00001BFEDC390C09&show_window=yes&new_window=yes`

**Comment by skotzko** [comment]
Any thoughts / update on this?

**Comment by drgrib** [comment]
I'm honestly probably not going to have time to implement this personally as this is a feature I would never personally use in my current work. If that changed, I would implement it.

**Comment by drgrib** [comment]
If it's very important for your workflow and you don't use the current link system, you probably will want to just take the time to edit your personal copy of the workflow to have a regex Utilities > Replace node after the output of the current link node that strips out the title markup and just leaves the x-callback URL.

**Comment by skotzko** [comment]
Went ahead and did that. Thank you for the suggestion. Rather than change the existing pathway, I duplicated it and added the "paste untitled link" as a third option, with its own modifier keys.

I made the edits in the Alfred workflow editor, wasn't sure how to contribute those back in code. But I [exported the modded workflow to my Dropbox](https://www.dropbox.com/s/bh9xtc8cdavbm7t/Bear%20-%20w%20plain%20note%20link.alfredworkflow?dl=0) if you want to incorporate or fold it in. 

Note: in this mod, I changed the modifier key for the original link + title to be shift + option, and the plain link (no title) to just be shift, since I use that much more frequently.
